evalml/pipelines/multiseries_regression_pipeline.py

```diff
@@ -163,16 +163,45 @@ def predict_in_sample(
             objective,
             calculating_residuals,
         )
+        unstacked_predictions = unstacked_predictions[
+            [
+                series_id_target
+                for series_id_target in y_train_unstacked.columns
```

Comment:
Looks like we'd want to use `self.series_id_target_names` instead of `y_train_unstacked.columns` to mirror above, right?

Reply:
When unstacked, the column names include the target name with the series ID values (`target|series_a`, `target|series_b`) rather than just the series ID values (`series_a`, `series_b`)

Reply:
I must be missing something then - don't we save `self.series_id_target_names = y_unstacked.columns` (where y_unstacked is the training y) above? Or does it change somewhere without me noticing?

Reply:
Oh true, this one is on me 🤦‍♂️. Somehow I misread this as just the series ID values. I'll update it to use `self.series_id_target_names` instead for parity.